site2/docs/getting-started-standalone.md

```diff
@@ -1,278 +1,146 @@
 ---
 id: getting-started-standalone
-title: Set up a standalone Pulsar locally
+title: Run a standalone Pulsar cluster locally
 sidebar_label: "Run Pulsar locally"
 ---
 
-For local development and testing, you can run Pulsar in standalone mode on your machine. The standalone mode includes a Pulsar broker, the necessary [RocksDB](http://rocksdb.org/) and BookKeeper components running inside of a single Java Virtual Machine (JVM) process.
-
-> **Pulsar in production?**  
-> If you're looking to run a full production Pulsar installation, see the [Deploying a Pulsar instance](deploy-bare-metal.md) guide.
-
-## Install Pulsar standalone
-
-This tutorial guides you through every step of installing Pulsar locally.
-
-### System requirements
-
-Currently, Pulsar is available for 64-bit **macOS**, **Linux**, and **Windows**. To use Pulsar, you need to install 64-bit JRE/JDK.
-For the runtime Java version, see [Pulsar Runtime Java Version Recommendation](https://github.com/apache/pulsar/blob/master/README.md#pulsar-runtime-java-version-recommendation) according to your target Pulsar version.
```

Comment:
I think the Java version requirement is also important in this doc. Are there other places in the documentation where the java version is specified?

Reply:
You're right. I should revert changes to this info.

Reply:
Updated at e486f28cf9064570a628170d1fdd5a5bfd83124a. Although, I think we should inline the version instead of jumping to GitHub README. But I don't have a clear idea of this improvement. Can do it as a follow-up.

Reply:
We can just tell the user to use Java version 17 for this quickstart guide. And junmp to the GitHub README for more detail.

Reply:
Yep. Let me think of it for a bit. This can be affected when it comes to versioned docs. I'd prefer to write information _just_ for the current version, and when the content is archived to be a historical version, it still holds. But we have a complex site building setup so I must think of it for a bit, lol.